src/containers/BooksList.js

```diff
@@ -2,25 +2,40 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
-import { removeBook } from '../actions/index';
+import CategoryFilter from '../components/CategoryFilter';
+import { removeBook, changeFilter } from '../actions/index';
 
 const BooksList = props => {
-  const { books, handleRemoveBook } = props;
+  const {
+    books, filter, handleRemoveBook, handleFilterChange,
```

Comment:
- [ ] **[OPTIONAL]** Same here we can destruct objects inside function arguments for brevity.